### DISCUSSION TITLE
How to shift career to coding related side.

### DISCUSSION THREAD

**Discussion by uwatude** [community/community#38862] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Sorry this is not very technical related question but..

I've been working on IT Technical support role for long time and I would like to shift my career to coding related side as much as possible. (HTML, CSS or Python etc)
However I'm not sure how to shift my career to there even I study them and test with github when I have time.
Of course I do make test sites and apps, and then add them to my resume as my portfolio. 

Could you please advise what I should do to shift my career to there  (how to start my career)? 
Maybe any recommended certificates?
I'm not sure how to grab hiring teams attention for such positions.

Much appreciated if there are any tips!!

So far I keep applying such positions but no luck so far.

Best regards,

**Comment by jfullstackdev** [comment]
I also did the same thing, from teaching to coding, well I still teach here in the Philippines but in the College level already

I got the job when I impressed the Directing Manager of my demo site, which is my personal project. Never realized I would be using that to get the job.

if you really want to get the coding job, search online in your country what are the roles that are the most sought, here in the Philippines, it's Full Stack Developer because you know, they want to keep efficiency in coding as possible

and there are languages and frameworks they are up to, the most common two are Java and its framework Spring Boot and the other one, which is my role now, is PHP and its framework Laravel

and since you are from IT, yup, you have the advantage more than me, because you are already in the industry and have the exposure in coding 

and also here in the Philippines, just after the peak of pandemic, Accenture hired a lot of software engineers, like 5,000 openings, as they are expanding and the good thing is that, you don't need to be an actual developer, they will train you for six months ! 

but I'm not in Accenture, my company is different, right after you were hired you will be assigned in an actual project, coding right away

**Comment by uwatude** [comment, marked as answer]
Thank you for your input. Much much appreciated! :)

May I know what sort of project you've done to impress "the Directing Manager"? (Java and PHP related?)
The most sought role wise, I think same here in Japan. 

Also, so far, I've been using Udemy to learn coding stuff.
How did you start learning them?

I really feel changing "0" to "1" is so difficult that is starting new stuff like this. (Hard to stand at a coding career starting point)
Hopefully I can make it sometime soon, but I kinda lost what to do to get their attention. 
E.g. Making them impressed by my resume and proceed to interviews.  (So far I barely make it at this point)

Again thank you for your great tips!
The Tips from a person like you is very helpful to me! 
Salamat po!

**Reply by jfullstackdev** [reply]
do an actual personal project while you are learning, I have here on GitHub my actual project a lightweight web server coded in Java

and during the interview, since it's a demo, I was asked the usual coding questions, the core, like Arrays, HTTP, how is data being passed from the user to the database etc.

mostly they are seeking for full stack here, because budgeting is a constraint if they will be hiring front-end, backend, DevOps to launch the production, so you will be needing to have a framework other than the language and tech stack, like PHP Laravel, Java Spring Boot

and it was actually too tough even to get an online interview, I passed I think more than 200 applications with just only 10 companies contacting for an initial interview and I got the call at my current company through LinkedIn, so it helps too

**Reply by uwatude** [reply]
Thank you very much for your tips again. :) 
Your every words help me a lot. 

I just took a look at your page, which is awesome to me. 

Before you find your current job, have you also done freelance work that is coding related before?
Or you jumped into your current position straight forward from way different industry? (I'm assuming this is the one but for just in case..)

Also have you taken any online courses that are like several days courses with paying expensive tuition?
I'm trying to figure out what way is the best to learn faster since I started to sense learning by myself is very tough.
